fiftyone/utils/aws.py

```diff
@@ -0,0 +1,127 @@
+"""
+Utilities for working with
+`Amazon Web Services <https://aws.amazon.com/>`
+S3 storage buckets.
+
+| Copyright 2017-2021, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+import logging
+import multiprocessing
+import multiprocessing.dummy
+import os
+from urllib.parse import urlparse
+
+import boto3
+import botocore
+
+import fiftyone.core.utils as fou
+
+import eta.core.utils as etau
+
+
+logger = logging.getLogger(__name__)
+
+
+def download_from_s3(
```

Comment:
How about `download_public_s3_files()` instead to clarify the scope of this method?